doc/developer/project-management.md

```diff
@@ -0,0 +1,122 @@
+# Project Management
+
+At Materialize, we have a
+[unified EPD roadmap](https://github.com/orgs/MaterializeInc/projects/43).
+The roadmap describes all prioritized projects across our EPD teams,
+including: Product Teams, Functional Teams, Product Design, Developer
+Experience, Education, and Analytics. Each roadmap item is marked with
+a priority (“Now,” “Next,” or “Later”) that indicates the timeline upon
+which the item will be picked up, worked on, and eventually delivered.
+
+Each project marked “Now” must be owned by a single Directly Responsible
+Individual (DRI). The DRI for a project can be any member of the larger
+EPD team. The DRI is responsible for driving the project to completion,
+and also for keeping the status of the project visible and up to date
+at all times. The DRI for any given project can change over time, as
+long as the handoff is publicly documented.
+
+The unified EPD roadmap is created by EPD management once a quarter.
+For the most part, projects are picked up and prioritized from the
+existing backlog of issues. In some cases, new projects will be spun
+up to address an urgent and important need that is not already
+represented in the backlog.
+
+The EPD roadmap is not set in stone. Although we only holistically
+revise the roadmap once a quarter, prioritizations of individual
+projects can and should change as we learn new information. EPD
+management is responsible for ensuring that we are working on the
+right things each week in a recurring Roadmap Review. During this
+review, the priority and staffing of any individual project can
+be updated.
+
+The rest of this document describes how individual projects are
+managed at Materialize, broken down into four interconnected phases:
+identification, triage, discovery, and delivery.
+
+## Identification
+
+Fundamentally, there are two types of issues we create to identify
+potential work: bugs and problems. Bugs report a defect in an
+existing feature. Bugs can be reported by internal team members
+and external users, and should be created in the repository that
+contains the buggy code wherever possible.
+
+Problems, on the other hand, can identify potential improvements
+as well as net-new features. Because of this, they can vary greatly
+in both size and complexity. They can be as small and as scoped as
+adding a simple SQL function to Materialize, or as large and as
+unbounded as making sure that users can sufficiently observe Materialize’s
+performance. Both types of problems are important to solve in order to
+ensure the overall success of Materialize as a product.
+
+By default, all created issues will end up in a Materialize-wide
+backlog. Issues created in our public Github repository will be
+visible outside of our team.
+
+## Triage
+
+The purpose of the Triage phase is to ensure we’re working on the
+most urgent and important problems at all times.
+
+Newly reported bugs will be triaged most frequently. `P0` or `release-blocker`
```

Comment:
Is triage the process that assigns these priorities? (Does this imply that all issues are triaged within 24h in case one of them is P0?)

Reply:
No, typically the issue filer assigns the `p0` or `release-blocker` label. The goal of the triage process is to ensure that every issue filed issue against our main repositories gets processed (in a timely manner).

Tried to clarify in 30970683ac.